File: src/main/java/ftc/shift/sample/repositories/SubscribeRepository.java
package ftc.shift.sample.repositories;

import ftc.shift.sample.models.Group;
import ftc.shift.sample.models.Subscription;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SubscribeRepository extends JpaRepository<Subscription, Long> {

}

File: src/main/java/ftc/shift/sample/repositories/SubscribeRepository.kt
package ftc.shift.sample.repositories

import ftc.shift.sample.models.Subscription
import ftc.shift.sample.models.User
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SubscribeRepository : JpaRepository<Subscription, Long> {
    fun findByGroupId(groupId: Long): List<Subscription>
    fun findByUserId(userId: Long): List<Subscription>
}